static/app/utils/replays/replayReader.tsx

```diff
@@ -39,6 +40,11 @@ import {
 } from 'sentry/utils/replays/types';
 import type {ReplayError, ReplayRecord} from 'sentry/views/replays/types';
 
+interface ClipWindow {
+  endTimestamp: number;
+  startTimestamp: number;
```

Comment:
can we indicate what unit these expect? i'm assuming they're expected to be milliseconds since the epoch?
```suggest
  endTimestampMs: number;
  startTimestampMs: number;
```